Python/dp/Matrix-Chain-Multiplication.py

```diff
@@ -0,0 +1,41 @@
+import sys
```

Comment:
Add a beginner pitch enclosed in a multiline string detailing about the Code/Algorithm